src/client/clientmap.cpp

```diff
@@ -378,7 +378,7 @@ void ClientMap::updateDrawList()
 
 		// Block meshes are stored in blocks where all coordinates are even (lowest bit set to 0)
 		// Add them to the de-dup set.
-		shortlist.emplace(block_coord.X & ~1, block_coord.Y & ~1, block_coord.Z & ~1);
+		shortlist.emplace(block_coord.X / CLIENT_CHUNK_SIZE * CLIENT_CHUNK_SIZE, block_coord.Y / CLIENT_CHUNK_SIZE * CLIENT_CHUNK_SIZE, block_coord.Z / CLIENT_CHUNK_SIZE * CLIENT_CHUNK_SIZE);
```

Comment:
at this point there really should be a `roundToSize` or `roundToVolume` helper

Reply:
Oh... And I need to fix that comment too :)

Reply:
Do you mean a general ROUND_TO_SIZE macro:
```
// v and r must long or int
#define ROUND_TO_SIZE(v, r) ((v) / (r) * (r))
```
(This does not make it more readable, IMHO)

Or specific macro/function for this rounding for meshing?